### PR TITLE
CI: execution strategy changed

### PR DESCRIPTION
**Pull request by 650elx**
Since now we're executing tests against release branches (as opposed to main branch), execution strategy has to be changed.

### CODE REVIEW

File: .github/workflows/run-ws.yml
Comment (asein-sinch) [inline .github/workflows/run-ws.yml:1]: What is the reason to delete this file?

**Comment by asein-sinch** [comment]
The CI is failing when trying to clone a private repo. Is your PAT expired?

**Comment by 650elx** [comment]
> The CI is failing when trying to clone a private repo. Is your PAT expired?

Yeah; there was a temporary issue. It's fixed

File: .github/workflows/run-ws.yml
Comment (650elx) [reply .github/workflows/run-ws.yml:1]: I've disabled White Source GH action a while a go.
I was simply not using it. Sorry for not mentioning that in the description of PR.

**Review by asein-sinch — APPROVED** [review]

**Comment by asein-sinch** [comment]
> Yeah; there was a temporary issue. It's fixed
Was it an issue on GH side or did you have to perform an action regarding the GH_PAT secret?

**Comment by 650elx** [comment]
> > Yeah; there was a temporary issue. It's fixed
> 
> Was it an issue on GH side or did you have to perform an action regarding the GH_PAT secret?

GH_PAT. I have to change that. I'll add ticket to the planning.

**Pull request merged**